fluent-package/manage-fluent-repositories.sh

```diff
@@ -70,7 +70,14 @@ case $COMMAND in
 	   DRYRUN_OPTION=""
 	fi
 	for target in $TARGETS; do
-	    command="aws s3 sync $DRYRUN_OPTION --delete $FLUENT_RELEASE_DIR/5/$target s3://packages.treasuredata.com/5/$target --profile $FLUENT_RELEASE_PROFILE"
+	    case $FLUENT_RELEASE_DIR in
+		*lts)
+		    command="aws s3 sync $DRYRUN_OPTION --delete $FLUENT_RELEASE_DIR/5/$target s3://packages.treasuredata.com/lts/5/$target --profile $FLUENT_RELEASE_PROFILE"
+		    ;;
+		*)
+		    command="aws s3 sync $DRYRUN_OPTION --delete $FLUENT_RELEASE_DIR/5/$target s3://packages.treasuredata.com/5/$target --profile $FLUENT_RELEASE_PROFILE"
+		    ;;
+	    esac
```

Comment:
I see...
This change looks a little drastic.

I don't think it's a problem anyway, since this script is very simple.
This is fine for now.

Reply:
Added usage explanation.